setup.py

```diff
@@ -1,14 +1,17 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # read the contents of your README file
+import sys
 from pathlib import Path
-
 from setuptools import find_packages, setup
 
 this_directory = Path(__file__).parent
 long_description = (this_directory / "README.md").read_text()
 
-lint_deps = ["flake8", "mypy==1.8.0", "types-requests"]
+if sys.version_info >= (3, 8):
+    lint_deps = ["flake8", "mypy==1.8.0", "types-requests"]
+else:
+    lint_deps = ["flake8", "mypy==1.4.1", "types-requests"]
```

Comment:
Btw if it becomes too painful in the future,I think we could also just lint/typecheck on recent python version